src/extras/random.jl

```diff
@@ -10,22 +10,3 @@ function rand(rng::AbstractRNG, S::Type{DoubleFloat{T}}) where {T<:IEEEFloat}
 
     DoubleFloat(T(5.421010862427522e-20 * f), T(5.421010862427522e-20 * rf))
 end
-
-rand(::Type{DoubleFloat{T}}) where {T<:AbstractFloat} =
-    rand(Random.GLOBAL_RNG, DoubleFloat{T})
-
-for T in (:Float64, :Float32, :Float16)
-  @eval begin
-        
-    function rand(::Type{DoubleFloat{$T}}, n::Int)
-        rs = Array{DoubleFloat{$T}}(undef, n)
-        for idx=1:n
-            rs[idx] = rand(DoubleFloat{$T})
-        end
-        return rs
-    end
-        
-  end
-end
-
-rand(::Type{DoubleFloat}, n::Int) = rand(DoubleFloat{Float64}, n)
```

Comment:
I deleted this line as this is not consistent, and `rand(Double64, n)` will do the same while being shorter. But I'm fine to revert that part, I may miss something regarding the usefulness of this method.